src/main/java/com/sforce/cd/apexUnit/client/fileReader/ApexManifestFileReader.java

```diff
@@ -170,4 +138,43 @@
 		}
 		return testClassesAsArray;
 	}
+	
+	private void insertIntoTestClassesArray(String strLine, ArrayList<String> testClassList){
```

Comment:
Yes, agreed. I would like to add the different argument types and the return type.